app/routes/devices.js

```diff
@@ -9,6 +9,7 @@ var devicesSchemas = require('../schemas/devices');
 router.get('/', state.onlyWhenReady, auth.authorize(), devicesCtrl.list);
 router.get('/:id', state.onlyWhenReady, auth.authorize(), celebrate(devicesSchemas.get), devicesCtrl.get);
 router.put('/:id', state.onlyWhenReady, auth.authorize(), celebrate(devicesSchemas.update), devicesCtrl.update);
+router.post('/', auth.authorize(), celebrate({ body: devicesSchemas.create }), devicesCtrl.create);
```

Comment:
In the future, we must also enforce this operation could be done only when the state is `ready` (take the above endpoints as examples when using `state.onlyWhenReady` rule). Could you add a TODO comment?

Reply:
Done!